.pre-commit-config.yaml

```diff
@@ -19,17 +19,26 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.0.1
     hooks:
-      - id: trailing-whitespace
       - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: check-yaml
+      - id: pretty-format-json
 
   - repo: https://github.com/PyCQA/isort
     rev: 5.8.0
     hooks:
       - id: isort
-        args: [--settings-path, ./pyproject.toml]
+        name: imports
 
   - repo: https://github.com/pre-commit/mirrors-yapf
     rev: v0.31.0
     hooks:
       - id: yapf
-        args: [--parallel, --in-place]
```

Comment:
Do we not need args anymore? why?

Reply:
see: https://github.com/kornia/kornia/pull/1038#discussion_r642515193